### PR TITLE
Avjain refactoring

### PR DESCRIPTION
**Pull request by AvJain**
Changes - 
`Player.py` updated to inherit from ChatEmbed 
`MusicQueue.py` updated to inherit from ChatEmbed 
`Reporter.py` updated to inherit from ChatEmbed and setup function deprecated

### CODE REVIEW

File: src/reporter.py
Comment (Exsects) [inline src/reporter.py:32]:
```suggestion
```

File: src/reporter.py
Comment (Exsects) [inline src/reporter.py:33]:
```suggestion
```

Comment (Exsects) [inline src/reporter.py:33]:
```suggestion
```

@AvJain @borisliao I believe this function is also deprecated since it's not being used anywhere else outside.

**Review by borisliao — DISMISSED** [review]

File: src/reporter.py
Comment (Exsects) [inline src/reporter.py]:
```suggestion
```

Looks like this function can also be remove since the logout button is deprecated.

Comment (AvJain) [reply src/reporter.py]: done

File: src/reporter.py
Comment (Exsects) [inline src/reporter.py]: Is there a reason we can just delete this entire setup function?

**Review by Exsects — APPROVED** [review]
Looks good! Works on local.

**Review by borisliao — APPROVED** [review]
Wait for @bakhoon to approve before merge.

**Review by bakhoon — APPROVED** [review]

**Pull request merged**
